### PR TITLE
DOC:Added Examples To `combine_pvalues` and `fhtoffset`

### PR DESCRIPTION
**Pull request by devbijay**
Added Examples To `combine_pvalues` and `fhtoffset` docstrings

Part of #7168

### CODE REVIEW

**Comment by ilayn** [comment]
Hi @devbijay Thank you for the examples. Two things would make this PR even better; 

- Please refrain from putting issue numbers to commit messages because they bark at the original issue and create a bit too much noise, like the following 
image
it is certainly not a big deal but tiny peace of mind for us : ) 
- And since you are only doing changes to documentation we don't need to run the whole test suite so in future please could you add `[skip actions] [skip cirrus]` to the end of your commit message as per https://docs.scipy.org/doc/scipy/dev/contributor/continuous_integration.html#skipping

File: scipy/fft/_fftlog.py [hunk 1]
```diff
@@ -334,6 +334,16 @@ def fhtoffset(dln, mu, initial=0.0, bias=0.0):
...
 
+    Examples
+    --------
+    >>> dln = 0.1
+    >>> mu = 2.0
+    >>> initial = 0.5
+    >>> bias = 0.0
+    >>> offset = fhtoffset(dln, mu, initial, bias)
+    >>> print(offset)
+    0.539805138227967
```
Comment (mdhaber) [inline scipy/fft/_fftlog.py:345]: The output needs to match that produced by SciPy.
```suggestion
    0.5454581477676637
```

**Review by mdhaber — CHANGES_REQUESTED** [review]
In the future, please submit unrelated changes in separate PRs. Including them together can only slow down review (e.g. the different changes may require different reviewers, so the whole PR is delayed until all reviewers have finished).

Please also include all imports explicitly and ensure that the output matches that produced by SciPy.

File: scipy/stats/_stats_py.py [hunk 2]
```diff
@@ -9629,6 +9629,18 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
...
             The combined p-value.
 
+    Examples
+    --------
+    >>> pvalues = [0.1, 0.05, 0.02, 0.3]
+    >>> result = combine_pvalues(pvalues, method='fisher')
+    >>> print(result.pvalue)
+    0.21907768033406858
+
+    >>> weights = [1, 2, 3, 4]
```
Comment (mdhaber) [inline scipy/stats/_stats_py.py:9639]:
```suggestion
   When the individual p-values carry different weights, consider Stouffer's
   method.

    >>> weights = [1, 2, 3, 4]
```

**Review by mdhaber — APPROVED** [review]
Will merge if CI (CircleCI only) passes. Please review detailed feedback, though, @devbijay.

File: scipy/stats/_stats_py.py [hunk 3]
```diff
@@ -9629,6 +9629,24 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
...
+    Suppose we wish to combine p-values from four independent tests
+    of the same null hypothesis using Fisher's method (default).
+
+    >>> from scipy.stats import combine_pvalues
+    >>> pvalues = [0.1, 0.05, 0.02, 0.3]
+    >>> combine_pvalues(pvalues)
+    SignificanceResult(statistic=20.828626352604235, pvalue=0.007616871850449092)
+
+   When the individual p-values carry different weights, consider Stouffer's
+   method.
```
Comment (mdhaber) [inline scipy/stats/_stats_py.py:9643]:
```suggestion
    When the individual p-values carry different weights, consider Stouffer's
    method.
```

**Pull request merged**